packages/changed-elements-react/src/common/FeedbackButton.scss

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+.button {
```

Comment:
These simple class names have high probability to collide with other CSS classes in consuming applicaton. Just use top-level `itwin-cer--feedback-btn` class name and address nested elements with CSS selectors.